app/assets/stylesheets/content/_modal.sass

```diff
@@ -91,6 +91,12 @@ $ng-modal-image-width:   $ng-modal-image-height
 
   #modal-sorting
     min-height: 120px
+    .form--row
+      overflow: visible
```

Comment:
@0xF013 `overflow: visible` is already set on `.form--row`. Is this overriden somewhere?